Robinhood/Robinhood.py

```diff
@@ -308,374 +348,491 @@ def get_historical_quotes(
+    def extended_hours_market_value(self):
+        """
+            Wrapper for portfolios
 
+            Returns:
+                (float): `extended_hours_market_value` value
         """
+
         try:
             return float(self.portfolios()['extended_hours_market_value'])
+
         except TypeError:
             return None
 
-    def last_core_equity(self):
-        """wrapper for portfolios
 
-        get `last_core_equity` value
+    def last_core_equity(self):
+        """
+            Wrapper for portfolios
 
+            Returns:
+                (float): `last_core_equity` value
         """
+
         return float(self.portfolios()['last_core_equity'])
 
-    def last_core_market_value(self):
-        """wrapper for portfolios
 
-        get `last_core_market_value` value
+    def last_core_market_value(self):
+        """
+            Wrapper for portfolios
 
+            Returns:
+                (float): `last_core_market_value` value
         """
+
         return float(self.portfolios()['last_core_market_value'])
 
-    def market_value(self):
-        """wrapper for portfolios
 
-        get `market_value` value
+    def market_value(self):
+        """
+            Wrapper for portfolios
 
+            Returns:
+                (float): `market_value` value
         """
+
         return float(self.portfolios()['market_value'])
 
-    def order_history(self):
-        """wrapper for portfolios
 
-        get orders from account
+    def order_history(self):
+        """
+            Wrapper for portfolios
 
+            Returns:
+                (:obj:`dict`): JSON dict from getting orders
         """
+
         return self.session.get(self.endpoints['orders']).json()
 
-    def dividends(self):
-        """wrapper for portfolios
 
-        get dividends from account
+    def dividends(self):
+        """
+            Wrapper for portfolios
 
+            Returns:
+                (:obj: `dict`): JSON dict from getting dividends
         """
+
         return self.session.get(self.endpoints['dividends']).json()
 
-    ##############################
-    # POSITIONS DATA
-    ##############################
+
+    ###########################################################################
+    #                           POSITIONS DATA
+    ###########################################################################
 
     def positions(self):
-        """Returns the user's positions data."""
+        """
+            Returns the user's positions data
+            
+            Returns:
+                (:object: `dict`): JSON dict from getting positions
+        """
+
         return self.session.get(self.endpoints['positions']).json()
 
+
     def securities_owned(self):
         """
-        Returns a list of symbols of securities of which there are more
-        than zero shares in user's portfolio.
+            Returns a list of symbols of securities of which there are more
```

Comment:
80 char warning, 100 char line break.  This might be too restrictive?

Reply:
Grammar change --> might be more readable/concise